articles/libraries/lock/v10/api.md

```diff
@@ -209,3 +211,32 @@ lock.on("authenticated", function(authResult) {
 - `authorization_error`: emitted when authorization fails. Has the error as its only argument.
 - `hash_parsed`: _Note that this is a low level event for advanced use cases, and `authenticated` and `authorization_error` should be preferred when possible._ Every time a new Auth0Lock object is initialized in redirect mode (the default), it will attempt to parse the hash part of the URL, looking for the result of a login attempt. After that, this event will be emitted with `null` if it couldn't find anything in the hash. It will be emitted with the same argument as the `authenticated` event after a successful login or with the same argument as `authorization_error` if something went wrong. This event won't be emitted in [popup mode](/libraries/lock/v10/popup-mode), because in popup mode, there is no need to parse the URL's hash part.
 
+
+## resumeAuth()
+
+If you set the [auth.autoParseHash](/library/lock/v10/customization) option to `false`, you'll need to call this method to complete the authentication flow. This method is useful when you're using a client-side router that uses a `#` to handle urls (angular2 with `useHash`, or react-router with `hashHistory`).
```

Comment:
This throws a 404 https://auth0-docs-content-pr-4118.herokuapp.com/docs/library/lock/v10/customization